### ISSUE TITLE
Henkaku 3.68 error c1-2758-2

### ISSUE THREAD

**Issue by CurrentlyRetro** [TheOfficialFloW/h-encore#88]
Hello everyone. I am having a little problem with my Vita and I don't want to restore unless it can be helped.

When I try to delete any bubble, I get the (C1-2758-2) error which prevents deletion. Another thing is that when I restart my Vita, the database keeps updating for like 30 seconds. Can someone please direct me in the direction I should go?

If I have to I'll do a factory reset and do the hack again since I feel comfortable doing it now. Any help would be appreciated 🙏

**Comment by Masters110** [comment]
Sd2vita?  If yes its not installed correctly

**Comment by CurrentlyRetro** [comment]
No. I don't have a SD to vita adapter. I just the official memory card that is 64 GB. I did install it correctly. I was using Henkaku 3.68  with no problems for about 2 months then all of a sudden I can't delete bubbles and every time I restarted, I get a database update for 30 seconds. I know I did something wrong but can't quite figure it out. I started learning more mods on youtube for the Vita and I'm sure I did it. 

I took the memory card out reinserted it and everything was gone after that. I just reinstalled Henkaku H-encore via the QCMA installer and now my Vita is fine again but I have to redownload all my games and homebrews😔. On the bright side I guess, all of my plugins were still in the tai folder. How odd. Again sorry for being noobish if I seem that way. I just get a thrill of playing with the Vita. It is my favorite next to the PSP.
